nss-day-cohort-50/rare-client-rambunctious-rattlers
# Ct post dynamic route 2

## Changes in: ApplicationViews.js
Added dynamic route path for post details view

## Changes in: PostList.js
Added key and link tag to title line of return statement

## Changes in: PostDetail.js
Changed category dot notation to ‘label’

## To Test
While in post list view in browser, click title and confirm that view changes to post details
